lib/ruby_lsp/ruby_lsp_rails/document_symbol.rb

```diff
@@ -161,8 +161,12 @@ def handle_all_arg_types(node, message)
             name = arg_receiver.full_name if arg_receiver.is_a?(Prism::ConstantPathNode)
```

Comment:
Unrelated to your PR changes, but I just realized that these can also be merged, similar to [here](https://github.com/Shopify/ruby-lsp-rails/blob/cecdf903f1ddbc15b2a9b5aa88e71fcde1f04ca6/lib/ruby_lsp/ruby_lsp_rails/document_symbol.rb#L223).